images/installer-origin-release/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# This Dockerfile is used to create the image quay.io/coreos/origin-release:bazel used in CI
+# to build smoke tests.
+
+FROM docker.io/openshift/origin-release:golang-1.10
+
+COPY bazel-copr-vbatts.repo /etc/yum.repos.d/
+
+RUN yum install -y bazel gcc-c++ libtool && \
+    yum clean all
+
+RUN wget https://github.com/openshift/origin/releases/download/v3.10.0-rc.0/openshift-origin-client-tools-v3.10.0-rc.0-c20e215-linux-64bit.tar.gz && \
```

Comment:
Looks like [they cut 3.10.0 proper on 2018-08-02][1].  Can we use that instead?

We can also save some local files here with:

```Dockerfile
RUN curl -s https://github.com/openshift/origin/releases/download/v3.10.0/openshift-origin-client-tools-v3.10.0-dd10d17-linux-64bit.tar.gz | tar -xz -C /usr/bin --strip-components 1 '*/oc'
```

[1]: https://github.com/openshift/origin/releases/tag/v3.10.0